### DISCUSSION TITLE
Docker compose not setting env issue

### DISCUSSION THREAD

**Discussion by apurvjha123** [ToolJet/ToolJet#10355] [Q&A]
this is the error  
`postgrest                                 | 13/Jul/2024:22:15:06 +0000: {"code":"PGRST000","details":"connection to server at \"pg-2502b5e6-lakeai-87ad.i.aivencloud.com\" (172.21.0.3), port 20367 failed: Connection refused\n\tIs the server running on that host and accepting TCP/IP connections?\n","hint":null,"message":"Database connection error. Retrying the connection."}
postgrest                                 | 13/Jul/2024:22:15:06 +0000: Attempting to reconnect to the database in 16 seconds...`

when i provide correct value
`# Create .env from this example file and replace values for the environment.
# The application expects a separate .env.test for test environment configuration
# Get detailed information about each variable here: https://docs.tooljet.com/docs/setup/env-vars

TOOLJET_HOST=http://localhost:80
LOCKBOX_MASTER_KEY=0efc6d0769e13c20dde6cc6250fa40baf3be76aa85a6d50da7a448d0d352bc91
SECRET_KEY_BASE=8e715bb55b43f1be5c2a9b6b15bd80ece619695b2931b4e0b25137833505e95ea4ba5e56b088e5b1dfd0d59c209e4b60300726447c9913d08023ca79b03f0c7f

# DATABASE CONFIG
# ORM_LOGGING=all
PG_DB=defaultdb
PG_USER=avnadmin
PG_HOST=pg-2502b5e6-lakeai-87ad.i.aivencloud.com
PG_PORT = 20367
PG_PASS=Im putting correct password here# postgres database password
`

**Comment by apurvjha123** [comment]
can trying do docker install refencing from https://docs.tooljet.com/docs/contributing-guide/setup/docker/

**Reply by asemandeveloper** [reply]
When I run `docker compose up`, I get the following error message!

`failed to copy: httpReadSeeker: failed open: failed to do request: Get "https://production.cloudflare.docker.com/registry-v2/docker/registry/v2/blobs/sha256/ea/eae53aac4f7dcbc45e032f77a48e611703810feaa35611adf31c54d86f5f9e8d/data?verify=1722867685-d3vx9lS6HIx8Q%2FxngYv61KZhLZk%3D": EOF`

What is the way?

**Comment by PriteshKiri** [comment]
Hey @apurvjha123,

Can you please share the server container logs? This will help us debug the issue better.
